pydynamo_brain/pydynamo_brain/ui/topMenu.py

```diff
@@ -200,6 +205,16 @@ def registerSmart(self):
         self._local().smartRegisterImages(self.stackWindow.windowIndex)
         self.redraw()
 
+    def radiiEstimator(self):
```

Comment:
style: optional, but perhaps 'multipleRadiiEstimator', to echo the style of 'singleRadiusEstimator'